admin_pages/events/Events_Admin_List_Table.class.php

```diff
@@ -515,4 +537,31 @@ public function column_actions(EE_Event $item)
             'div'
         );
     }
+
+
+    /**
+     * Helper for adding columns conditionally
+     *
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    private function addConditionalColumns()
+    {
+        $event_category_count = EEM_Term::instance()->count(
+            [['Term_Taxonomy.taxonomy' => 'espresso_event_categories']]
```

Comment:
There is a constant `EEM_CPT_Base::EVENT_CATEGORY_TAXONOMY` that could be used eh

Reply:
ooo ya good catch, I'll use the constant.